pkg/cmd/server/origin/run_components.go

```diff
@@ -361,7 +361,7 @@ func (c *MasterConfig) RunDeploymentImageChangeTriggerController() {
 // RunSDNController runs openshift-sdn if the said network plugin is provided
 func (c *MasterConfig) RunSDNController() {
 	oClient, kClient := c.SDNControllerClients()
-	controller, err := factory.NewMasterPlugin(c.Options.NetworkConfig.NetworkPluginName, oClient, kClient)
+	controller, err := factory.NewMasterPlugin(c.Options.NetworkConfig.NetworkPluginName, oClient, kClient, c.EtcdHelper)
```

Comment:
We have converted NetNamespace as virtual resource backed by Namespace. So using client API will hit Namespace object and it populates NetNamespace. For migration, we need to access persisted NetNamespace from etcd. 
I don't know if we have other APIs/mechanism to achieve this without using etcdHelper interface.
@liggitt ^^